src/domain/profile/modal/editProfileModal.module.scss

```diff
@@ -0,0 +1,52 @@
+@import 'mixins';
+@import 'variables';
+@import 'layout';
+
+.email {
+  label {
+    font-weight: bold;
+  }
+  p {
+    margin-top: 0.25em;
+  }
+}
+
+.buttonsWrapper {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
```

Comment:
have you check the responsive?

Reply:
Well, the buttons are so small that they fit next to each other also on small screens. Need to check with Anniina if side-by-side is ok.

Reply:
cool, np then